examples/turtlesim/BUILD.bazel

```diff
@@ -0,0 +1,87 @@
+load("@com_github_mvukov_rules_ros2//ros2:cc_defs.bzl", "ros2_cpp_binary")
+load("@com_github_mvukov_rules_ros2//ros2:interfaces.bzl", "cpp_ros2_interface_library")
+load("//third_party/qt:qt.bzl", "ros_qt_cc_library")
+
+cpp_ros2_interface_library(
+    name = "cpp_turtlesim_interface",
+    deps = [
+        "//turtlesim/action:turtlesim",
+        "//turtlesim/msg:turtlesim",
+        "//turtlesim/srv:turtlesim",
+    ],
+)
+
+ros_qt_cc_library(
+    name = "turtle",
+    src = "src/turtle.cpp",
+    hdr = "include/turtlesim/turtle.h",
+    includes = [
+        "include",
+    ],
+    deps = [
+        ":cpp_turtlesim_interface",
+        "@qt//:qt_core",
+        "@qt//:qt_gui",
+        "@ros2_common_interfaces//:cpp_geometry_msgs",
+        "@ros2_rclcpp//:rclcpp",
+        "@ros2_rclcpp//:rclcpp_action",
+    ],
+)
+
+ros_qt_cc_library(
```

Comment:
Once the `ament_index_cpp::get_package_share_director` issues is fixed, the images are to be added here, too.